staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -182,16 +216,19 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, invocatio
 	} else {
 		newVersionedObject, err = o.GetObjectCreater().New(attr.VersionedKind)
 		if err != nil {
-			return apierrors.NewInternalError(err)
+			return false, apierrors.NewInternalError(err)
 		}
 	}
+
+	changed := !apiequality.Semantic.DeepEqual(attr.VersionedObject, newVersionedObject)
```

Comment:
newVersionedObject is empty here... I'm not sure I understand what this is checking

Reply:
Fixing this by moving it down below `jsonSerializer.Decode`. I'll need to add a test case for this.